AGENTS.md

```
@@ -0,0 +1,205 @@
+
+## Common commands
+
+### Python setup
+```pwsh
+python -m venv .venv
+.\.venv\Scripts\Activate.ps1
+pip install -r requirements.txt
+```
+
+Install dev tooling:
+```pwsh
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)